.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @bboreham
```

Comment:
There seems to be an extra space here :)

Reply:
Thanks. Fixed.